admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -54,15 +54,34 @@
       {/if}
 
       {if isset($maintenance_mode) && $maintenance_mode == true}
+        {capture name="title"}
+          <p class="text-left text-nowrap">
+            <strong>{l s='Your store is in maintenance mode.' d='Admin.Navigation.Notification'}</strong>
+          </p>
+          <p class="text-left">
+              {l s='Your visitors and customers cannot access your shop while in maintenance mode.' d='Admin.Navigation.Notification'}
+          </p>
+          <p class="text-left">
+              {l s='To manage the maintenance settings, go to Shop Parameters > Maintenance tab.' d='Admin.Navigation.Notification'}
+          </p>
+          {if isset($maintenance_allow_admins) && $maintenance_allow_admins}
+            <p class="text-left">
+              {l s='Admins can access the store front-end without storing their IP.' d='Admin.Navigation.Notification'}
+            </p>
+          {/if}
+        {/capture}
         <div class="component hide-mobile-sm" id="header-maintenance-mode-container">
           <a class="link shop-state"
              id="maintenance-mode"
              data-toggle="pstooltip"
              data-placement="bottom"
              data-html="true"
-             title="<p class=&quot;text-left&quot;><strong>{l|escape s='Your shop is in maintenance.' d='Admin.Navigation.Notification'}</strong></p><p class=&quot;text-left&quot;>{l|escape s='Your visitors and customers cannot access your shop while in maintenance mode.%s To manage the maintenance settings, go to Shop Parameters > Maintenance tab.' sprintf=['<br />'] d='Admin.Navigation.Notification'}</p>" href="{$link->getAdminLink('AdminMaintenance')|escape:'html':'UTF-8'}"
+             title="{$smarty.capture.title|htmlspecialchars}"
+             href="{$link->getAdminLink('AdminMaintenance')|escape:'html':'UTF-8'}"
```

Comment:
```suggestion
             href="{$link->getAdminLink('AdminMaintenance')}"
```

Reply:
Escaping in the backoffice is still necessary. I checked